src-django/authentication/tests/test_signup.py

```diff
@@ -17,10 +19,36 @@ def test_new_user_can_signup(self):
             'accept_tos': True
         }
 
-        response = self.client.post('/auth/signup', fields)
+    def test_new_user_can_signup(self):
+        response = self.client.post('/auth/signup', self.valid_fields)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
         r = json.loads(response.content)
         assert_true(r['success'])
         assert_true(r['token'] is not None)
         assert_equals(len(r['errors']), 0)
+
+    def test_new_user_is_unconfirmed(self):
+        response = self.client.post('/auth/signup', self.valid_fields)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        user = User.objects.get(username='admin')
+        assert_false(user.profile.is_email_confirmed)
+
+    def test_email_confirmation_key_is_created(self):
```

Comment:
Since you're doing a valid `POST` here as well, maybe you want to set the `BROKER_BACKEND` to `memory`? I think that's why the tests are failing.